.circleci/config.yml

```diff
@@ -27,10 +27,14 @@ jobs:
         default: "1.23.3"
         type: string
       platform:
-        default: ubuntu2204
+        default: ubuntu_2204
         # This could be of type executor, but then conversion to a string is wonky :(
         type: string 
     executor: <<parameters.platform>>
+    environment:
+      # The GitHub artifacts list is flat, so all the parameters we need have to be specified
+      # in the name of the file.
+      MODULE_NAME: ngx_http_akita_module_amd64_<<parameters.platform>>_<<parameters.version>>.so
```

Comment:
Why change the names of the executors? This makes it hard to parse the module name: some underscores delimit pieces of the module name, while others are now part of the platform name.

Reply:
I wanted to add a delimiter because the platform name is now going to be automatically generated by akita-cli.

But ".", the obvious choice, is not allowed as part of an executor name so it's either "-" or "_", or not have a delimiter like I initially chose.

The right thing long-term is to decouple the executor name from what the resulting artifact is labelled, but the only way I can figure to do that is add a script to generate the name, since CircleCI's configuration doesn't seem to have a way to perform any sort of mapping.

Reply:
I was more worried about ambiguity between the OS version and the nginx version.  :(

Examples are amd64_ubuntu_2204_1.23.2 and amd64_amzn_2_1.18.0, so I agree it's not great but I don't think ubuntu2004_1.23.2 and amzn2_1.18.0 are a lot better.

Reply:
Yeah, neither is great. But at least with the Amazon example, it clearer that the 2 is associated with amzn when it's "amzn2". This said, if we don't expect to be parsing these names, then it doesn't really matter which we choose.